WcaOnRails/spec/spec_helper.rb

```diff
@@ -84,4 +84,15 @@
   # test failures related to randomization by passing the same `--seed` value
   # as the one that triggered the failure.
   Kernel.srand config.seed
+
+  config.before(:suite) do
+    %w(Countries Continents Events Rounds Formats teams).each do |table|
+      # -- Uncomment to regenerate --
+      # ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+
+      if ActiveRecord::Base.connection.execute("SELECT COUNT(*) FROM #{table}").first == [0]
+        load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
Working with strings makes this clumsier than it has to be

    [Countries Continents Events Rounds Formats teams].each do |model_class|
      # -- Uncomment to regenerate --
      # model_class.delete_all

      if model_class.count == 0
        load "#{Rails.root}/db/seeds/#{model_class.name}.seeds.rb"
      end
    end

A few other concerns:

* I don't think using `/db/seeds/` is a great approach here. That's intended to be used with `rake db:seed`, and executing code by using `load` is pretty gross. I'd make a `SeedTestDb` class and call it from here. Not sure where it should live, but it probably doesn't matter much.

* I worry about how this gets updated when the seed data changes. Let's say the `Formats` seed data changes. How does that get propagated to every developer working on their home machine? The `formats` table will not be empty, so it won't trigger on that.